hugo.yaml

```diff
@@ -171,6 +171,9 @@ params:
     docker-compose-v2: |
       `docker-compose` is deprecated. For details, see
       [Migrate to Compose V2](https://docs.docker.com/compose/migrate/).
+    browser-instrumentation: |
+      Client instrumentation for the browser is **experimental** and mostly **unspecified**. If you are interested in 
+      helping out, get in touch with the [Client Instrumentation SIG](https://docs.google.com/document/d/16Vsdh-DM72AfMg_FIt9yT9ExEWF4A_vRbQ3jRNBe09w/edit)
```

Comment:
Oops, missed that: this sentence should end with a period.

Reply:
let's have a follow up PR then for the period.

Reply:
See #3134.